src/language/en-US.json

```
@@ -611,7 +611,7 @@
     "loading": "Loading...",
     "locked_mosaic": "locked mosaic",
     "login": "Login",
-    "login_to_symbol_account": "Login to your Symbol account",
+    "login_to_symbol_account": "Login to your Symbol profile",
```

```suggestion
    "login_to_symbol_account": "Log in to your Symbol profile",
```

Maybe now is a good time to start fixing this mistake: "Log in" is a verb, whereas "login" is a noun:
https://english.stackexchange.com/questions/5302/log-in-to-or-log-into-or-login-to